proto/buf/alpha/module/v1alpha1/module.proto

```diff
@@ -20,6 +20,27 @@ import "buf/alpha/breaking/v1/config.proto";
 import "buf/alpha/lint/v1/config.proto";
 import "google/protobuf/timestamp.proto";
 
+enum HashKind {
+  HASH_KIND_UNSPECIFIED = 0;
+  HASH_KIND_SHAKE256 = 1;
+}
+
+// Hash represents a hash function's value.
+message Hash {
+  // kind describes the hash algorithm. e.g. "SHAKE256"
+  HashKind kind = 1;
```

Comment:
Should these be called `Digest` and `DigestType` to match the names in the manifest package? Also, do we really need two different representations -- plain Go types/structs _and_ protos? Why aren't the generated protos enough?

Reply:
Updated for symmetry with the manifest package.